code/game/machinery/computer/voxresearch_shuttle.dm

```diff
@@ -0,0 +1,145 @@
+var/global/datum/shuttle/voxresearch/voxvoxresearch_shuttle = new(starting_area = /area/shuttle/voxresearch/station)
+
+/datum/shuttle/voxresearch
+	name = "Research Shuttle"
+	can_link_to_computer = LINK_FREE
+	req_access = 0
+
+/datum/shuttle/voxresearch/initialize()
+	.=..()
+	add_dock(/obj/structure/docking_port/destination/voxresearch/station)
+	add_dock(/obj/structure/docking_port/destination/voxresearch/outpost)
+
+/obj/machinery/computer/shuttle_control/voxresearch/New() //Main shuttle_control code is in code/game/machinery/computer/shuttle_computer.dm
+	link_to(voxvoxresearch_shuttle)
+	.=..()
+
+//code/game/objects/structures/docking_port.dm
+
+/obj/structure/docking_port/destination/voxresearch/station
+	areaname = "voxresearch dock"
+
+/obj/structure/docking_port/destination/voxresearch/outpost
+	areaname = "voxresearch outpost"
+
+
+/**********************Shuttle Computer**************************/
```

Comment:
I have no idea why the mining shuttle, research shuttle, and now this all use the same exact code block.  The research shuttle (and this) should really be a subtype instead of 3 copies of identical code.  

I do not know how to do that and it was a struggle to even get my own shit to work.